constraints.md

```diff
@@ -109,55 +78,79 @@ INSERT INTO users (username) VALUES ('a');
 SELECT * FROM users;
 ```
 
-## Primary Key
+## UNIQUE KEY
 
-In TiDB, `PRIMARY KEY` constraints are checked lazily by default. By batching checks until when the transaction commits, TiDB is able to reduce network communication. For example:
+Depending on the transaction mode and the value of `tidb_constraint_check_in_place`, TiDB might check `UNIQUE` constraints [lazily](/transaction-overview.md#lazy-check-of-constraints). This helps improve performance by batching network access.
+
+For example:
+
+{{< copyable "sql" >}}
 
 ```sql
-mysql> CREATE TABLE t1 (id INT NOT NULL PRIMARY KEY);
-Query OK, 0 rows affected (0.01 sec)
+DROP TABLE IF EXISTS users;
+CREATE TABLE users (
+ id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+ username VARCHAR(60) NOT NULL,
+ UNIQUE KEY (username)
+);
+INSERT INTO users (username) VALUES ('dave'), ('sarah'), ('bill');
+```
 
-mysql> INSERT INTO t1 VALUES (1);
-Query OK, 1 row affected (0.00 sec)
+With the default of pessimistic locking:
```

Comment:
I think this could be misinterpreted: "the default pessimistic locking" implies that there could be more than one type of pessimistic locking. I am going to reject this edit.